neo/rawio/openephysrawio.py

```diff
@@ -273,23 +277,44 @@ def _parse_header(self):
         spike_channels = np.array(spike_channels, dtype=_spike_channel_dtype)
 
         # event file are:
-        #    * all_channel.events (header + binray)  -->  event 0
+        #    * all_channel.events (header + binary)  -->  event 0
+        #    * n_RhythmData-a.events (header + binary) --> event 0 (maybe a new naming convention? )
         # and message.events (text based)      --> event 1 not implemented yet
         event_channels = []
         self._events_memmap = {}
-        for seg_index, oe_index in enumerate(oe_indices):
-            if oe_index == 0:
-                event_filename = "all_channels.events"
-            else:
-                event_filename = f"all_channels_{oe_index + 1}.events"
-
-            fullname = os.path.join(self.dirname, event_filename)
-            event_info = read_file_header(fullname)
-            self._event_sampling_rate = event_info["sampleRate"]
-            data_event = np.memmap(fullname, mode="r", offset=HEADER_SIZE, dtype=events_dtype)
-            self._events_memmap[seg_index] = data_event
-
-        event_channels.append(("all_channels", "", "event"))
+        event_files = list(
+            [
+                event_file
+                for event_file in Path(self.dirname).glob("**/*.events")
+                if event_file.name != "messages.events"
+            ]
+        )
+        event_files.sort()
```

Comment:
This is the part I'm a little shaky on. If I sort and all the files are `xx_RhythmData-A.events` vs `xx_RhythmData-A.events_2` will this sort correctly?